health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CreateCampaign.js

```
@@ -86,6 +103,14 @@ const CreateCampaign = ({ hierarchyType, hierarchyData }) => {
       [name]: formData,
     }));
 
+    if (formData?.CampaignName) {
+      let temp = await fetchValidCampaignName(tenantId, formData);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'temp' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)